src/app/help/en/welcome.md

```diff
@@ -39,27 +37,24 @@ Below that is a list of recommended users to follow, a collection of other resou
 - <a href="#Password_Security">Password Security</a>
 - <a href="#Earning_on_Steemit">Earning on Steemit</a>
 
-### <a href="#Helpful_Posts_from_Steemit_Users">Helpful Posts from Steemit Users</a>
 ### <a href="#Users_to_Follow">Users to Follow</a>
 ### <a href="#Other_Resources">Other Resources</a>
-### <a href="#Live_Help">Live Help</a>
+### <a href="#Get_Help">Get Help</a>
 ### <a href="#Third_Party_References">Third Party References</a>
 
 ***
 
 ## To Do List
 <span></span>
-### <span id="Backup_your_password">1. Backup your password</span>
-
-Unlike centralized web services, **the Steem Blockchain has no account password recovery**.
+### <span id="Backup_your_password">1. Backup Your Password</span>
 
-You are entirely responsible for keeping your password, and keeping it secure.
+Unlike centralized web services, **it is not possible to recover lost passwords on the Steem blockchain**.
 
-Save your master key and keep it somewhere safe.
+You are entirely responsible for generating your password, saving it and keeping it secure.
 
-It is strongly recommended that you store an offline copy of your password somewhere safe in case of a hard drive failure or other calamity. Consider digital offline storage, such as a flash drive or burned CD, as well as printed paper. Use a safe deposit box for best redundancy. 
```

Comment:
Passed this through a friend who is a senior security consultant. Flash drives and burned CDs fall under digital offline storage. Secure cloud storage was recommended as an accessible alternative for mobile users, which encompasses a large portion of Steemit users. Safe deposit boxes are only accessible to a small user base and confusing.

Reply:
This paragraph is about offline storage. Cloud based storage is out of scope. Not to mention that if your e-mail will be compromised it also means that your secure cloud-based storage is no longer secure. Certain set of celebrities can tell you the difference between "secure cloud-based storage" and a safe deposit box. I cannot agree that they are accessible to a small user base. It's a common service available in banks and post offices. Also, that's just an example of offline storage for redundancy.
Real life scenario: a steemian with his house burned to the ground.

Reply:
I ran your concerns past the consultant. What they're suggesting is secure (private) cloud vs public cloud or cloud designed for basic storage. There is a major security report for the crypto industry coming out this summer I'm told. They're not recommending banks because its not an option in most parts of the world and safety deposit boxes can be seized. Relating the info I got on that.